packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -847,6 +856,9 @@ msgstr "Název e-shopu"
 msgid "EAN"
 msgstr "EAN"
 
+msgid "Each line contains exactly one valid regular expression. E-mails are sent to addresses which comply with at least one regular expression or to all addresses when no regular expression is provided. You can use <a href=\"https://regex101.com\">https://regex101.com</a> for validating regular expressions."
+msgstr "Každý řádek obsahuje přesně jeden platný regulární výraz. E-maily jsou odesílány na adresy, které vyhovují alespoň jednomu regulárnímu výrazu, nebo na všechny adresy, pokud není zadán žádný regulární výraz. Pro ověření platnosti regulárních výrazů můžete použít <a href=\"https://regex101.com\">https://regex101.com</a>."
+
```

Comment:
I debated with @henzigo on this topic for a while yesterday and came to the conclusion that I would be in favor of removing the master email. 

The functionality would be that there exists only the whitelist. If it is set up, emails are only sent to those addresses. If not set, all emails are sent.

The whitelist settings would be in the DB (see this issue). They can also be set in ENV variables, which override the settings in the DB (and whitelist administration show information). The `.env` files will contain the default whitelist setting, and only in the production stage the ENV variables will be set to empty during deployment, so that only all emails will be sent there.

The notification bar will show only whitelist settings.

Does it make sense?

Reply:
Reworked as suggested. `Mailer::send` which uses master email address was marked as deprecated in favor of new `Mailer::sendForDomain` method, which behave like suggested.